src/shared-components/EventForm/VisibilitySelector/VisibilitySelector.tsx

```
@@ -13,10 +14,15 @@ import styles from './VisibilitySelector.module.css';
 const VisibilitySelector: React.FC<InterfaceVisibilitySelectorProps> = ({
   visibility,
   setVisibility,
-  tCommon,
+  disabled = false,
 }) => {
+  const { t: tCommon } = useTranslation('common');
   return (
-    <fieldset className="mb-3">
+    <fieldset
+      className="mb-3"
+      role="radiogroup"
```

Biome (2.4.4)

[error] The HTML element fieldset is non-interactive and should not have an interactive role.

(lint/a11y/noNoninteractiveElementToInteractiveRole)